docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md

```diff
@@ -31,8 +28,10 @@ PORT or remove the -p flag if default port is used.
 
 Enable and start Gitea at boot:
 ```
-sudo systemctl enable gitea
-sudo systemctl start gitea
+sudo systemctl enable gitea && sudo systemctl start gitea
```

Comment:
I realy would just leave it as it is - I know you you like to give gitea users a hint about the `--now` function but  
but I think we should keep it simple (dont make changes to the `Enable and start Gitea at boot:` section)

beside this I think this are good chagnes

Reply:
-> do you have a reason to expose the `sudo systemctl enable gitea --now` in our docs?

Reply:
@6543 just shorcut to enable and start the services

Reply:
I personaly would lave it as it is now ...